src/scenic/syntax/compiler.py

```diff
@@ -184,6 +184,9 @@ def __init__(self) -> None:
         self.behaviorLocals: set = set()
         "Set of variable names on the local scope of the behavior"
 
+        self.inCompose: bool = False
+        "True if the transformer is processing scenario"
```

Comment:
Let's be more precise in this comment, since the flag isn't true inside the `setup` block, only the `compose` block.

Reply:
Fixed in 2c73958eb72971dcb12d86dd5329a5e1b35b039a